### ISSUE TITLE
Feature: Ability to Preview in app by entering Id and version

### ISSUE THREAD

**Issue by bhavincb** [NativeScript/playground-feedback#48]
feature: use playground app id and version number to preview inside mobile app.

**Comment by darind** [comment]
@bhavincb, could you please elaborate a bit more about this requirement? I am not quite sure to understand it quite right.

**Comment by bhavincb** [comment]
currently we can preview playground app by scanning the QR code to mobile playground app. but this is only way currently. i'm suggesting there should be an input field available on mobile playground app. in which we can enter playground project ID and then we can preview it.

**Comment by darind** [comment]
@bhavincb, you already can do that. For example try typing the following url in your mobile browser: https://play.nativescript.org/?template=play-vue&id=b9acUH and it should open the mobile version of the playground allowing you to directly browse the saved session.

**Comment by bhavincb** [comment]
i'm talking about android and ios native playground app. see the output of any playground demo by inserting it's id to native playground app.

**Comment by darind** [comment]
Yes, currently there's no way to insert only the `id` of a saved session, you need to type the full url or click on a link.

**Comment by bhavincb** [comment]
yup, that is the feature request i am asking.
